src/pipes/toFiat.ts

```diff
@@ -0,0 +1,18 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { TxFormatProvider } from '../providers/tx-format/tx-format';
+import { ConfigProvider } from '../providers/config/config';
+
+@Pipe({ name: 'toFiat' })
+export class ToFiatPipe implements PipeTransform {
+  private unitCode: string;
+
+  constructor(
+    private configProvider: ConfigProvider,
+    private txFormatProvider: TxFormatProvider
+  ) { 
+    this.unitCode = this.configProvider.get()['wallet']['settings'].unitCode;
```

Comment:
configProvider.get() return a typed object. Not necessary to use ['xx'].

Reply:
Got it!